styles.css

```diff
@@ -21,3 +21,33 @@ header {
 .logo-emphasis {
   font-weight: 700;
 }
+
+main {
+  display:flex;
```

Comment:
Looks like you're missing some spaces after your properties!

Reply:
Yep. I totally did. I'll get that up to date! Thanks!